code/modules/mob/living/carbon/human/human.dm

```diff
@@ -10,6 +10,8 @@
 	var/datum/species/species //Contains icon generation and language information, set during New().
 	var/embedded_flag	  //To check if we've need to roll for damage on movement while an item is imbedded in us.
 
+    var/fartCooldown = 20 SECONDS
```

Comment:
instead of a tab/indent, you used spaces here

Reply:
Yeah I was editing it on the phone. Fixed it now